packages/meditrak-server/src/tests/apiV2/surveyResponse.test.js

```diff
@@ -77,7 +82,7 @@ describe('surveyResponse endpoint', () => {
     const [{ survey }] = await buildAndInsertSurveys(models, [
       {
         id: surveyId,
-        code: 'TEST_SURVEY_' + generateId(), // prevent test cross-pollination
```

Comment:
Why this change?

Reply:
Ah there is no cross-pollination anyway since each test file runs in isolation, and the total char length was >30, so I shortened it.

Reply:
Gotcha